organization/_posts/2021/2021-04-01-coordination.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "HSF Coordination Meeting #205, 1 April 2021"
+layout: plain_toc
+---
+
+Present:
+Graeme Stewart,
+Teng Jian Khoo,
+Andrea Valassi,
+Paul Laycock,
+Serhan Mete,
+Eduardo Rodrigues,
+Meirin Oan Evans,
+Kyle Knoepfel,
+Michel Jouvin,
+Benedikt Hegner,
+Ben Morgan,
+Attila Krasznahorkay,
+Allison Hall,
+Krzysztof Genser,
+Sudhir Malik,
+Liz Sexton-Kennedy,
+Chris Jones,
+David Lange,
+Dorothea Vom Bruch,
+Host Severini,
+Josh McFayden,
+Kevin Pedro,
+Michael Villanueva,
+Mike Kirby,
+Philippe Canal,
+Daniel Elvira
+
+Apologies: Andi Salzburger, Pere Mato, Efe Yazgan
```

Comment:
Retroactive apologies from me (holiday weekend here). Thanks for the minutes though, this is where they're useful!